README.md

```diff
@@ -14,71 +14,81 @@ A curated list of awesome things related to [Chart.js](https://www.chartjs.org)
+  [deferred](https://github.com/chartjs/chartjs-plugin-deferred) | Defers initial chart update until chart scrolls into viewport | ✔ |
+  [dragdata](https://github.com/chrispahm/chartjs-plugin-dragdata) | Lets users drag data points on the chart | ✔ | ✔
+  [hierarchical](https://github.com/sgratzl/chartjs-plugin-hierarchical) | Adds support for hierarchical categorical scales that can be collapsed, expanded, and focused | ✔ | ✔
+  [regression](https://github.com/pomgui/chartjs-plugin-regression) | Calculate and draw statistical linear, exponential, power, logarithmic, and polynomial regressions (trend lines) | ✔ |
+  [rough](https://github.com/nagix/chartjs-plugin-rough) | Draws charts in a sketchy, hand-drawn-like style using Rough.js | ✔ |
+  [stacked100](https://github.com/y-takey/chartjs-plugin-stacked100) | Draws 100% stacked bar chart | ✔ | ✔
+  [streaming](https://github.com/nagix/chartjs-plugin-streaming) | Adds support for live streaming data | ✔ | ✔
+  [style](https://github.com/nagix/chartjs-plugin-style) | Provides styling options such as shadow, bevel, glow or overlay effects | ✔ |
+  [waterfall](https://github.com/everestate/chartjs-plugin-waterfall) | Enables easy use of waterfall charts | ✔ |
+  [zoom](https://github.com/chartjs/chartjs-plugin-zoom) | Enables zooming and panning on charts | ✔ | ✔
 
 In addition, many plugins can be found on the [npm registry](https://www.npmjs.com/search?q=chartjs-plugin-).
 
 ## Adapters
 
-- [date-fns](https://github.com/chartjs/chartjs-adapter-date-fns) - date-fns adapter
-- [luxon](https://github.com/chartjs/chartjs-adapter-luxon) - Luxon adapter
-- [moment](https://github.com/chartjs/chartjs-adapter-moment) - Moment.js adapter
+  Name | Description | Chart.js v2 | Chart.js v3
+  ---- | ----------- | :--: | :--:
+  [date-fns](https://github.com/chartjs/chartjs-adapter-date-fns) | date-fns adapter | ✔ | ✔
+  [luxon](https://github.com/chartjs/chartjs-adapter-luxon) | Luxon adapter | ✔ | ✔
+  [moment](https://github.com/chartjs/chartjs-adapter-moment) | Moment.js adapter | ✔ | ✔
 
 ## Integrations
 
 ### JavaScript
 
-- [ember-cli-chart](https://github.com/aomran/ember-cli-chart) - Ember CLI
-- [lwcc](https://github.com/SalesforceLabs/LightningWebChartJS) - Lightning Web Component
-- [ng2-charts](https://github.com/valor-software/ng2-charts) - Angular v2+
-- [omi-chart](https://github.com/Tencent/omi/tree/master/packages/omi-chart) - Omi
-- [react-chartjs-2](https://github.com/jerairrest/react-chartjs-2) - React
-- [vue-chartjs](https://github.com/apertureless/vue-chartjs/) - Vue.js
+  Name | Description | Chart.js v2 | Chart.js v3
+  ---- | ----------- | :--: | :--:
+  [ember-cli-chart](https://github.com/aomran/ember-cli-chart) | Ember CLI | ✔ |
+  [lwcc](https://github.com/SalesforceLabs/LightningWebChartJS) | Lightning Web Component | ✔ |
+  [ng2-charts](https://github.com/valor-software/ng2-charts) | Angular v2+ | ✔ | ✔
```

Comment:
Since the V3 for ng2-charts is only available with the `@next` tag I think its better to keep it on V2 only since it might give confusion if people just try to install it nromally and use it together with v3